packages/sdk-core/src/modules/database/BacktraceDatabase.ts

```diff
@@ -16,7 +18,17 @@ import {
     ReportBacktraceDatabaseRecord,
 } from './model/BacktraceDatabaseRecord.js';
 
-export class BacktraceDatabase implements BacktraceModule {
+type BacktraceDatabaseEvents = {
```

Comment:
can we keep this class clean and move it to another file?

Reply:
why? it's directly correlated with this class and this class only

Reply:
Database configuration also. If we're making an excuse just for events, we will end up in the really bad state for this file.

Reply:
moved to another file